HealthcareAnalytics/HealthcareAnalytics/Views/Employees/Index.cshtml

```diff
@@ -61,3 +61,12 @@
 }
 
 </table>
+
+<ul class="pagination">
+    @for (int i = 1; i <= ViewBag.Total / ViewBag.ResultsPerPage; i++)
```

Comment:
If ViewBag.Total == 15 and ViewBag.ResultsPerPage is 10, how many will show?  I'm curious because it might default to an integer type and only show 1 instead of 2.  Have you tested this scenario?